app/controllers/users_controller.rb

```diff
@@ -1,3 +1,9 @@
 class UsersController < ApplicationController
-  expose :users, -> { User.all }
+  expose :user
+  expose_decorated(:users, -> { User.all })
```

Comment:
I highly recommend you to not use a `decent exposure` in any context.
Yep, It's much simpler but it will take the code out of your control some day.